adapters/webhdfs-adapter/src/test/java/com/connexta/replication/adapters/webhdfs/WebHdfsClientTest.java

```diff
@@ -255,6 +256,25 @@ public void testUpdateResourceWithNoUpdates() throws URISyntaxException {
     assertThat(adapter.updateResource(updateStorageRequest), is(false));
   }
 
+  @Test
+  public void testUploadedResourceMatchesOriginalResource() throws IOException, URISyntaxException {
+    String fileId = "123456789";
+    String testName = "testresource";
+
+    Calendar calendar = Calendar.getInstance();
+    long creationTime = calendar.getTimeInMillis();
+
+    String originalFilename = String.format("%s_%s.txt", fileId, creationTime);
+
+    // when a file is created with ID fileId
+    CreateStorageRequest createStorageRequest =
+        generateTestStorageRequest(fileId, testName, new Date(creationTime));
+
+    adapter.createResource(createStorageRequest);
+
+    verifyFileContents(originalFilename, "my-data");
```

Comment:
It might be helpful to add a comment here about where the "my-data" string is coming from.

Reply:
I'm replacing the string with a class constant